nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/exhibition/manifestations/ExhibitionMentionInPublication.java

```diff
@@ -0,0 +1,75 @@
+package no.unit.nva.model.instancetypes.exhibition.manifestations;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.model.time.Instant;
+import no.unit.nva.model.time.Time;
+import nva.commons.core.JacocoGenerated;
+
+import java.util.Objects;
+
+public class ExhibitionMentionInPublication implements ExhibitionProductionManifestation {
+
+    public static final String TITLE_FIELD = "title";
+    public static final String ISSUE_FIELD = "issue";
+    public static final String DATE_FIELD = "date";
+    public static final String OTHER_INFORMATION_FIELD = "otherInformation";
+
+    @JsonProperty(TITLE_FIELD)
+    private final String title;
+    @JsonProperty(ISSUE_FIELD)
+    private final String issue;
+    @JsonProperty(DATE_FIELD)
+    private final Time date;
+    @JsonProperty(OTHER_INFORMATION_FIELD)
+    private final String otherInformation;
+
```

Comment:
Mangler felt for sider som ogsp ligger i scenario PR
![bilde](https://user-images.githubusercontent.com/6313468/222438922-656fcfce-2022-4df3-adb9-d790d8d6ebce.png)